ground/src/main/res/layout/loi_card_item.xml

```diff
@@ -57,6 +56,7 @@
         tools:text="Map the terrain" />
       <androidx.constraintlayout.utils.widget.ImageFilterView
         android:id="@+id/loiIcon"
+        visible="@{!TextUtils.isEmpty(loi.caption)}"
```

Comment:
Can we eventually move this logic to the ViewModel? That might require an extra class to contain the card view states, but it might be worth it for better testability.

Reply:
+1

Reply:
Filed as a separate bug